packages/mui-material/src/Button/Button.js

```diff
@@ -295,9 +296,23 @@ const ButtonEndIcon = styled('span', {
   ...commonIconStyles(ownerState),
 }));
 
+const addPositionDataAttr = (buttonGroupButtonContext) => {
```

Comment:
Let's not use abbreviations

```suggestion
const addPositionDataAttributes = (buttonGroupButtonContext) => {
```